scripts/pushArgos.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+const util = require('util');
+const glob = require('fast-glob');
+const fse = require('fs-extra');
+const chunk = require('lodash/chunk');
+const childProcess = require('child_process');
+
+const execFileAsync = util.promisify(childProcess.execFile);
+
+async function exec(command, args) {
+  const options = {
+    cwd: process.cwd(),
+    env: process.env,
+    stdio: 'pipe',
+    encoding: 'utf-8',
+  };
+
+  const results = await execFileAsync(command, args, options);
+  return results.stdout;
```

Comment:
Could you just return it so that we don't confuse it later? When I read `exec` I expect the same signature as nodes's built-in. I understand that we don't need `stderr` since that's available from the rejected promise as well. But let's not try to write custom wrappers around well-documented node built-ins that we need to mentally unpack everytime we work on this script.